chartfx-math/src/main/java/de/gsi/math/MathBaseFast.java

```diff
@@ -69,17 +69,17 @@ public static float sinDeg(float a) {
     }
 
     private static void init() {
-        modulusRad = (int) (2.0 * Math.PI * precision);
+        modulusRad = (int) (2.0 * java.lang.Math.PI * precision);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.PI' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)